tools/exp/filter/filter.go

```diff
@@ -0,0 +1,154 @@
+package filter
+
+import (
+	"errors"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/mitchellh/copystructure"
+	"golang.org/x/exp/maps"
+)
+
+// errUnexpected is the error that is returned when an unexpected error occurs.
+var errUnexpected = errors.New("unexpected filtering error")
+
+// supportedServiceTypes is a pseudo-constant map of supported service types.
+var supportedServiceTypes = func() map[string]struct{} {
+	return map[string]struct{}{
+		"cassandra":                       {},
+		"clickhouse":                      {},
+		"datadog":                         {},
+		"elasticsearch":                   {},
```

Comment:
have we deprecated elasticsearch already? maybe it should be double checked

Reply:
`elasticsearch` is supported in Terraform at the moment